client/js/main.ts

```diff
@@ -201,59 +259,113 @@ function loadAttendees (filter: string = queryField.value, checkedIn: string = c
 	status.textContent = "Loading...";
 
 	let tag: string = tagSelector.value;
-	qwest.get("/api/search", {
-		"q": filter,
-		"tag": tag,
-		"checkedin": checkedIn
-	}).then((xhr, response: IAttendee[]) => {
+
+	// Get checked question options
+	let checked: string[] = [];
+	let checkedElems = document.querySelectorAll("#question-options input:checked") as NodeListOf<HTMLElement>;
+	for (let i = 0; i < checkedElems.length; i++) {
+		checked.push((<HTMLInputElement>checkedElems[i]).value);
```

Comment:
Don't need a type assertion here if you make the NodeList of HTMLInputElements as commented above